### DISCUSSION TITLE
gcd function

### DISCUSSION THREAD

**Discussion by ethanolboi** [microsoft/vscode-cpptools#11656] [General]
![image](https://github.com/microsoft/vscode-cpptools/assets/125242504/4af63c9e-e7ce-4ce7-adae-e345fd27fe58)
As shown in the image above the complier can't find the gcd function. Please can someone help me? I've already spent many many hours in VS code trying to fix these small problems. I'm about to lose my mind

**Comment by sean-mcmanus** [comment]
This was answered in https://github.com/microsoft/vscode-cpptools/issues/11655
